### DISCUSSION TITLE
What does the release schedule look like for ghidra?

### DISCUSSION THREAD

**Discussion by EmosewaMC** [NationalSecurityAgency/ghidra#5438] [General]
Keen on getting 10.3.1 since it has a bug fix which affects our project drastically (livelocking in quite a few spots) and i'm not up to date on how this organization updates their releases.  Thanks in advance!

**Comment by ryanmkurtz** [comment]
We don't share release schedules, but we recognize the severity of this bug and plan on getting something out soon.  

Also note that you may produce your own interim build of the `patch` branch for your organization to use, which is the development branch for the next patch release.

**Reply by EmosewaMC** [reply]
Thank you for the communication.  We'll consider using an interim build for now
